mozilla-central/setup

```diff
@@ -10,6 +10,9 @@ pushd $INDEX_ROOT
 if [ -d "gecko-dev" ]
 then
     echo "Found pre-existing gecko-dev folder; skipping re-download."
+    pushd $INDEX_ROOT/gecko-dev
```

Comment:
nit: I think pushd gecko-dev should be enough, and is consistent with the if condition.